selfdrive/ui/SConscript

```diff
@@ -47,7 +47,7 @@ asset_obj = qt_env.Object("assets", assets)
 qt_env.Program("soundd/_soundd", ["soundd/main.cc", "soundd/sound.cc"], LIBS=qt_libs)
 if GetOption('test'):
   qt_env.Program("tests/playsound", "tests/playsound.cc", LIBS=base_libs)
-  qt_env.Program('tests/test_sound', ['tests/test_runner.cc', 'soundd/sound.cc', 'tests/test_sound.cc'], LIBS=[base_libs])
+  qt_env.Program('tests/test_sound', ['tests/test_runner.cc', 'soundd/sound.cc', 'tests/test_sound.cc'], LIBS=qt_libs)
```

Comment:
Here, does including `soundd/sound.cc` a second time mean we re-use what we built above? If so we can keep `base_libs` and just wrap sound.cc in a `env.Object()`